### ISSUE TITLE
Possible "haddock" package name change for arch-repo

### ISSUE THREAD

**Issue by dedifferentiator** [haskell-infra/www.haskell.org#28]
[Haskell installation page](https://www.haskell.org/downloads/linux/) states that `haddock` package can be installed within `sudo pacman -S haddock`, although [there's no package](https://www.archlinux.org/packages/?sort=&q=haddock) in arch-repositories with such name. Instead there's a package with the name of [haskell-haddock-library](https://www.archlinux.org/packages/community/x86_64/haskell-haddock-library/), if that is the required package, maybe it is worth to change `haddock` to `haskell-haddock-library` on the site?

**Comment by TikhonJelvis** [comment]
Thanks for catching that.

Does the `haskell-haddock-library` package install the `haddock` *executable* as well as the library itself? I looked over the package page and it *seems* like it doesn't, but I'm not an Arch user and I can't really tell. (I also don't know if this is a real problem.)

**Comment by dedifferentiator** [comment]
I suppose package [is building](https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/haskell-haddock-library) something from [here](https://hackage.haskell.org/packages/archive/haddock-library/1.8.0/haddock-library-1.8.0.tar.gz) at the moment. I installed `haskell-haddock-library` and looks like the haddock binary exists.
```
~ haddock --version
Haddock version 2.24.0, (c) Simon Marlow 2006
Ported to use the GHC API by David Waern 2006-2008
```
```
~ haddock --help
Usage: haddock [OPTION...] file...

  -B DIR                                          path to a GHC lib dir, to override the default path
  -o DIR        --odir=DIR                        directory in which to put the output files
  -l DIR        --lib=DIR                         location of Haddock's auxiliary files
  -i FILE       --read-interface=FILE             read an interface from FILE
  -D FILE       --dump-interface=FILE             write the resulting interface to FILE
                --show-interface=FILE             print the interface in a human readable form
  -h            --html                            output in HTML (XHTML 1.0)
                --latex                           use experimental LaTeX rendering
                --latex-style=FILE                provide your own LaTeX style in FILE
                --mathjax=URL                     URL FOR mathjax
...
```

But I'm a bit new to haskell and I'm not sure what this library should exactly look like.

> (I also don't know if this is a real problem.)

Well, me too, but it's just a little confusing when you copy-paste a line with required packages from installation guide and then the package manager says that it cannot find one of them, so you have to look what's the matter, what is the name of the package, is it the package that I need etc, that slightly complicates things, especially for newcomers who don't know what they need :)

**Comment by TikhonJelvis** [comment]
Thanks for checking this! I think just having the binary is enough in this case.

> Well, me too, but it's just a little confusing when you copy-paste a line with required packages from installation guide and then the package manager says that it cannot find one of them

Absolutely. I'll put together a PR updating the command.

**Comment by gwils** [comment]
Closed by #33
